extension/popup/popupView.jsx

```diff
@@ -890,11 +912,40 @@ const SearchResultsContent = ({
     </div>
   );
 
+  const MusicCard = () => (
+    <div className="results-set">
+      {card.music.map(({ imgSrc, text, alt }) => (
+        <select
+          value={userSettings.musicService}
+          onChange={onMusicServiceChange}
+          onBlur={onMusicServiceChange}
+          className="music-list"
+        >
+          {userOptions.musicServices &&
+            userOptions.musicServices.map(musicOption => (
+              <option key={musicOption.name} value={musicOption.name}>
+                <div>
+                  <img
+                    className="results-image music-service-image"
+                    src={imgSrc}
+                    alt={alt}
+                  />
+                </div>
+                <div className="results-medium-text">{text}</div>
+              </option>
+            ))}
+        </select>
```

Comment:
I am not sure how to handle the options here @ianb , musicServices appear to be an array.